### PR TITLE
Fix docs and small edits

### PR DESCRIPTION
**Pull request by tmigot**
The Julia codes for adding a new backend were broken.

### CODE REVIEW

File: docs/src/index.md [hunk 1]
```diff
@@ -33,46 +33,46 @@ julia> performance_profile(PlotsBackend(), T, ["Solver 1", "Solver 2", "Solver 3
...
+                                    xlabel,
+                                    ylabel,
+                                    labels,
+                                    title,
+                                    logscale;
+                                    kwargs...)
+    #
+    # now produce the plot and return the plot object
+    #
+  end
```
Comment (abelsiqueira) [inline docs/src/index.md:59]:
```suggestion
  function performance_profile_plot(
      ::SomeNewPlotBackend,
      x_plot,
      y_plot,
      max_ratio,
      xlabel,
      ylabel,
      labels,
      title,
      logscale;
      kwargs...
  )
    #
    # now produce the plot and return the plot object
    #
  end
```
To follow our style.

File: docs/src/index.md [hunk 2]
```diff
@@ -33,46 +33,46 @@ julia> performance_profile(PlotsBackend(), T, ["Solver 1", "Solver 2", "Solver 3
...
+
+  function data_profile_plot(::SomeNewPlotBackend,
+                             T,
+                             xs,
+                             max_data,
+                             xlabel,
+                             ylabel,
+                             labels,
+                             title;
+                             kwargs...)
```
Comment (abelsiqueira) [inline docs/src/index.md:69]: Same here.

**Review by abelsiqueira — CHANGES_REQUESTED** [review]
Thanks for the update. I've noticed the function were not following our style, so I've suggested these changes.

File: docs/src/index.md (hunk 2 above)
Comment (tmigot) [reply docs/src/index.md]: Done. Thanks for the comments.

**Review by abelsiqueira — APPROVED** [review]

**Comment by dpo** [comment]
thank you

**Pull request merged**